src/AutoRest.CSharp/Common/Generation/Writers/ModelWriter.cs

```diff
@@ -425,6 +427,14 @@ public void WriteExtendableEnum(CodeWriter writer, EnumType enumType)
                         writer.Append($"public static {cs} {choice.Declaration.Name}").AppendRaw("{ get; }").Append($" = new {cs}({fieldName});").Line();
                     }
 
+                    // write ToSerial method, only write when the underlying type is not a string
+                    if (!enumType.IsStringValueType)
+                    {
+                        writer.Line();
+                        writer.Line($"internal {enumType.ValueType} {enumType.SerializationMethodName}() => _value;");
```

Comment:
nit: any reason we didn't just use ToInt32 vs ToSerialInt32?  Is there any actual difference between the two if they existed side by side?

Reply:
I choose this name to align the existing similar methods in fixed enum. If we have a fixed enum based on float, it will have an extension method `ToSerialSingle`. Therefore I am choosing the same pattern here.